js/packages/common/src/contexts/meta/loadAccounts.ts

```diff
@@ -305,10 +305,12 @@ export const loadAccounts = async (
         updateState('metadataByMint', metadata.info.mint, metadata);
         updateState('metadata', '', metadata);
 
-        state.metadataByAuction[auctionCache.info.auction] = [
-          ...auctionMetadata,
-          metadata,
-        ];
```

Comment:
What is metadata. what the rendering when metadata is put in the first index and the auction metadata spread as the tail.

Reply:
Yeah I tried switching them up first thinking that might be the fix, but it doesn't help.

`metadata` is the one that is effectively getting written here -- auctionMetadata is just a blank array every time I log it out (though I left it in there in case it's for some edge case I haven't found).

As long as the main NFT is being written in as item 0 in the array, it shows up correctly in the auction list.

The way it is in this commit is that effectively the main NFT gets written as item 0 and the participation NFT gets written as item 1.  The way it was before is that the participation NFT always got written as item 0, and then there was no item 1, etc.